Assignment2.md

```diff
@@ -6,53 +6,25 @@
 ## To set up and execute
 
 ### Download
-Download the ZIP file named `ay2324s1-course-assessment-g17-Assignment-2.zip` from [Assignment-2-Updated](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g17/releases/tag/Assignment-2-Updated).
-2. Unzip the file and navigate to the folder `ay2324s1-course-assessment-g17-Assignment-2`.
+Download the ZIP file named `ay2324s1-course-assessment-g17-Assignment-2-Updated.zip` from [Assignment-2-Updated](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g17/releases/tag/Assignment-2-Updated).
+2. Unzip the file and navigate to the folder `ay2324s1-course-assessment-g17-Assignment-2-Updated`.
 
 ### Include .env files
-3. Set up the environment for the question service, user service, and frontend by including the `.env` files for each, which have been submitted as: 
-`Assignment2-QuestionServiceEnv.txt`, 
-`Assignment2-UserServiceEnv.txt`, 
-`Assignment2-FrontendEnv.txt` respectively.
-Make sure they are `.env` files within the question service, user service, and frontend directories.
-
-For convenience, they have been placed here:
-
-`Assignment2-QuestionServiceEnv.txt`
-```
-MONGO_CONNECTION_STRING=mongodb+srv://sherylhci:wsY2vQnTjSStiIQ8@cluster0.kxz8d7c.mongodb.net/peerprep_app?retryWrites=true&w=majority
-PORT=8080
-ACCESS_TOKEN_SECRET="access_secret"
-REFRESH_TOKEN_SECRET="refresh_secret"
-FRONTEND_URL="http://localhost:3000"
-```
-
-`Assignment2-UserServiceEnv.txt`
-```
-DATABASE_URL="postgresql://postgres@localhost:5432/postgres?schema=public"
-PORT=8000
-ACCESS_TOKEN_SECRET="access_secret"
-REFRESH_TOKEN_SECRET="refresh_secret"
-FRONTEND_URL="http://localhost:3000"
-```
-
-`Assignment2-FrontendEnv.txt`
-```
-REACT_APP_BACKEND_URL=http://localhost:8080/api
-REACT_APP_USER_SERVICE_BACKEND_URL=http://localhost:8000/
-```
+3. Set up the environment for the question service, user service, and frontend by including the `.env` files for each, which have been submitted separately on CANVAS. Make sure they are `.env` files within the question service, user service, and frontend directories.
```

Comment:
this is nit picky but 
3. Set up the environment for `question-service`, `user-service`, and `frontend` by including the `.env` files for each, which have been submitted separately on CANVAS. Make sure they are `.env` files within the question service, user service, and frontend directories.